fluent/src/bundle.js

```diff
@@ -62,15 +62,7 @@ export default class FluentBundle {
     this._useIsolating = useIsolating;
     this._transform = transform;
     this._intls = new WeakMap();
-  }
-
-  /*
-   * Return an iterator over public `[id, message]` pairs.
-   *
-   * @returns {Iterator}
-   */
-  get messages() {
-    return this._messages[Symbol.iterator]();
+    this._dirty = new WeakSet();
```

Comment:
what's that? It doesn't seem to be used anywhere

Reply:
It's used in `createScope`, and then in `Pattern` to detect cyclic references.